KH1Client.py

```diff
@@ -0,0 +1,8 @@
+import ModuleUpdate
+import Utils
+from worlds.kh1.Client import launch
+ModuleUpdate.update()
+
+if __name__ == '__main__':
+    Utils.init_logging("KH1Client", exception_logger="Client")
```

Comment:
```suggestion
if __name__ == '__main__':
    import ModuleUpdate
    ModuleUpdate.update()

    import Utils
    Utils.init_logging("KH1Client", exception_logger="Client")

    from worlds.kh1.Client import launch
```
I understand wanting all the imports at the top, and the tools that complain if they're not at the top.

Calling `update` after the other imports is not very useful, because the imports will need the modules updated before the import if it makes any difference.
Also, the `init_logging` could be useful to help diagnose issues that could happen during import.

Luckily, nothing is needed if we're not running this script (as `__main__`), and the tools (that I know of) don't complain if the imports are inside the `if` block. So this is my recommendation for this script.